spec/lib/tmuxinator/window_spec.rb

```diff
@@ -208,6 +210,69 @@
     end
   end
 
+  describe "#synchronize" do
+    context "synchronization enabled" do
+      let(:synchronize) { true }
+
+      it "should have synchronization enabled" do
+        expect(window.synchronize).to be true
+      end
+    end
+
+    context "synchronization disabled" do
+      let(:synchronize) { false }
+
+      it "should have synchronization enabled" do
+        expect(window.synchronize).to be false
+      end
+    end
+
+    context "synchronization not specified" do
+      it "should have synchronization disabled" do
+        expect(window.synchronize).to be false
+      end
+    end
+  end
+
+  describe "#tmux_synchronize_panes" do
+    let(:project) { double(:project) }
+    let(:window) { Tmuxinator::Window.new(yaml, 0, project) }
```

Comment:
Thanks for the suggestion and your quick feedback, @adamstrickland . I added such a shared context with 5837d3e .